File: src/main/java/ru/dragonestia/apocalypse/item/ChipItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class ChipItem extends Item implements AdvItem {

    public ChipItem(Integer meta, int count) {
        super(ApocalypseID.CHIP, 0, count, "Chip");
    }

    @Override
    public String getRuName() {
        return "Микросхема";
    }

    @Override
    public String getIcon() {
        return "textures/items/upgrade/chip";
    }

    @Override
    public int getMaxStackSize() {
        return 4;
    }

}

File: src/ru/dragonestia/apocalypse/item/ChipItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item

class ChipItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.CHIP, 0, count, "Chip") {

    override fun getMaxStackSize() = 4

}